### PR TITLE
release-0.35.2-rc6  

### PR DESCRIPTION
**Pull request by Simpcyclassy**
## Description

release 0.35.2 rc6

## Related Issues

NA

## Testing

NA

## Merging

NA

### CODE REVIEW

**Review by danielhoherd — APPROVED** [review]
Standard version bump, this time to 0.35.2-rc6. LGTM.

We should not merge this until after https://github.com/astronomer/astronomer/pull/2277 has been cherry-picked to release-0.35

I've canceled CI here because there's no reason to run it for just a version bump.

**Comment by Simpcyclassy** [comment]
cherry-picked https://github.com/astronomer/astronomer/pull/2277

**Pull request merged**
